op-service/ctxinterrupt/signal-waiter.go

```diff
@@ -0,0 +1,52 @@
+package ctxinterrupt
```

Comment:
nit: replace hyphenated filename with underscores for consistency

Reply:
I avoid using underscores in file names due to [special treatment of underscores](https://pkg.go.dev/cmd/go#hdr-Build_constraints). Happy to change it if there's a preference for underscores tho.